opensfm/exif.py

```diff
@@ -222,6 +222,8 @@ def extract_sensor_width(self):
         if not mm_per_unit:
             return None
         pixels_per_unit = get_tag_as_float(self.tags, 'EXIF FocalPlaneXResolution')
+        if pixels_per_unit == 0:
+            return None
```

Comment:
```suggestion
        if pixels_per_unit <= 0:
            pixels_per_unit = get_tag_as_float(self.tags, 'EXIF FocalPlaneYResolution')
            if pixels_per_unit <= 0:
                return None
```
If for some reason `FocalPlaneXResolution` is `0` or is broken we should try `FocalPlaneYResolution` before we finally return `None`. Of course, this assumes, like before, that pixels are square shaped (which on the vast majority of photo sensors they are).